articles/rock-paper-scissor-online/index.md

```diff
@@ -0,0 +1,260 @@
+A variety of applications use Socket on the modern web. For example chat applications, real-time databases and online multiplayer games. All use Socket under the hood. 
+
+### Introduction
+This article focuses on building a web application which lets you play Rock Paper Scissor online with your friends. A player has to create a room and share the unique Room ID with the other player. The other player can join the same room with this unique Room ID. Once both players have joined the room, they could start playing!
+
+There are two parts to this application, 
+- Front-end: To keep things simple, the front-end is built on basic HTML, CSS, JS with jQuery.
+- Back-end: An Express server that uses Socket.io.
+
+### Prerequisites
+ 
+- HTML is the standard markup language for web page. Here's a useful [guide](https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML) for beginners.
+- CSS is style sheet language for improving the presentation of web pages. To learn more, check this [article](https://www.section.io/engineering-education/what-is-css/)
+- [BootStrap](https://getbootstrap.com/) is a popular HTML, CSS, JS library. An [article](https://getbootstrap.com/docs/4.0/getting-started/introduction/)  to get started.
+- [jQuery](https://jquery.com/) is a fast and feature-rich Javascript library. 
+- [Express.js](https://expressjs.com/) is a web application framework  for Node.js. Introductory article on Express.js: [link](https://www.section.io/engineering-education/express/).
+- [Socket.io](https://socket.io/) is a Real-Time Engine for Web. A helpful [article](https://www.section.io/engineering-education/understanding-socket/) to get started. This article covers the usage of Socket.io on both the server and client-side. These are essential to proceed further.
+
+
+
+### Starter Code
+
+In the interest of time and the intention to put more focus on Socket logic. Please download the starter code from this [Repository](https://github.com/HarishTeens/rps-online) on GitHub.  Follow the instructions as mentioned on README.md to set up the project on your local system.
+
+If you are only here for final version of the code, check out the [final branch](https://github.com/HarishTeens/rps-online/tree/final) of the same Repository.
+
+Here goes the file wise description of the Starter code,
+
+- ```public/index.html``` : A Home Page for the app with all the required UI Components. 
+- ```public/style.css``` : Styles for the entire application.
+- ```public/game.js``` : Client-Side JavaScript file to handle events.
+- ```app.js``` : Server Side NodeJS file for setting up the server.
+
+Note that the app is not functional yet, we need to write the necessary logic to make the app functional. Also, feel free to tweak the boiler code to match your style.
+
+There are two important files to focus on. One is  ```app.js``` that handles server-side logic. And the other is ```public/game.js``` that handles client-side logic.
```

Comment:
You only need one backtick for inline code elements. 

Join the next two sentences together. **logic and the other is**

Reply:
It felt a little better to have the file name highlighted though 😅  @louisefindlay23